form/form.go

```diff
@@ -99,6 +99,19 @@ type IOFiller struct {
 	// ShowDescriptions holds whether attribute descriptions
 	// should be printed as well as the attribute names.
 	ShowDescriptions bool
+
+	// GetDefault returns the default value for the given attribute,
+	// which must have been coerced using the given checker.
+	// If there is no default, it should return (nil, "", nil).
```

Comment:
I should "should" is right here - we're describing what you need to
do to make a valid GetDefault function. "does" would imply
that it always does that, which ain't necessarily so.